scripts/README.md

```diff
@@ -0,0 +1,11 @@
+# Scripts (too rule them all)
+
+The scripts in these folder are meant to help the user/contributor to get up and running with the application fast.
+
+`bootstrap` - will point you towards the setup you will need before picking up this application
```

Comment:
As with the main `README`, this script, and the ones below could be linked to - if that was wanted?